ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/fips/provision/cucushift-installer-rehearse-azure-ipi-fips-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-fips-provision
+  steps:
+  - chain: ipi-conf-azure
+  - chain: ipi-install
```

Comment:
I noticed there's one existing step [fips-check](https://github.com/openshift/release/tree/master/ci-operator/step-registry/fips-check) defined in the step-registry, do we want to add the ref here

Reply:
No, the https://github.com/openshift/release/tree/master/ci-operator/step-registry/fips-check comes from one bug(one related to real node is fips disabled on rhel nodes). The step will check whether fips enabled for rhel nodes before install ocp packages, if not, will refuse to do more installation.

Reply:
ok, thanks for the info